src/config/graphical-session/kde.md

```diff
@@ -17,3 +17,17 @@ instead.
 If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
+
+## Dolphin
+
+Dolphin is the default file manager of the KDE desktop environment. It can be
+installed on its own by installing the `dolphin` package, or it can be installed
+as part of the `kde5-baseapps` package.
+
+### Thumbnail Previews
+
+To have properly working thumbnail file previews, the packages `ffmpegthumbs`
+and `kdegraphics-thumbnailers` have to be installed. Enable previews in
+"Control" -> "Configure Dolphin" -> "General" -> "Previews" by checking the
+corresponding boxes. File previews will be shown for the selected file types
+after clicking "Preview" in Dolphin's toolbar.
```

Comment:
I'm a bit confused now. The necessary package is `ffmpegthumbs`, then, not `ffmpegthumbnailer`?

Reply:
Yes, according to Johnnynator. However, since `ffmpegthumbs` is independent of `kdegraphics-thumbnailers` and is only used to generate video thumbnails, it can't be added as a dependency.

Reply:
I see. Could you make that clear, then? Something like "To enable thumbnail file previews, install the `kdegraphics-thumbnailer` package. If you want video thumbnails, the `ffmpegthumbs` package is also necessary"